### ISSUE TITLE
Possible Cotton Crash

### ISSUE THREAD

**Issue by Diamondman121314** [CottonMC/Cotton#35]
**Describe the bug**
It seems my client has been crashing when it seemingly looks like I pass a certain amount of mods. I've been suggested to report this here. Now in the crash report, it shows both Inca and Cotton Resources, but seems to also include Cotton, as obviously being a dependency. I'm not sure which it would be in this case between the two. I do have about 95% of all working mods for Fabric which is somewhere around 300-350 so I'm not sure if this would be incompatibility between mods somewhere. I talked a lot about this in the #player-general channel of the Fabric Discord.

[Crash Log](https://gist.github.com/Diamondman121314/e767239a7015e15adbfeb9fd3c92ba59)

**To Reproduce**
1. Load the client with a decent number of mods, at least 150 from what I can tell.
2. See error

**Expected behavior** *Optional*
Client to load without any errors and to not crash.

**Screenshots** *Optional*
N/A

**Version of the (please complete the following information):**
 - Cotton 1.0.0-rc.2
 - Fabric API 0.3.0+build.200
 - Minecraft 1.14.4

**Additional context** *Optional*
If you need any more information or more testing to be done, I can do that.

**Comment by falkreon** [comment]
Looks like a bad Cotton Datapack + Cotton Logger impl, happening entirely within Cotton. This should help us a lot in tracking it down, thanks

**Comment by falkreon** [comment]
On second thought, peeking at the Inca source, it looks like it's built for Cotton 0.4.8 (which 1.0.0 contains breaking changes from). Does it crash without Inca?

**Comment by Diamondman121314** [comment]
It does still crash without Inca. It's the same error, just without the entire Inca section. If you need the crash report for that, I'll provide it.

**Comment by falkreon** [comment]
Nope. Just knocking out alternate explanations. The log you provided should be enough.

**Comment by Diamondman121314** [comment]
So from I can see, the recent update to Cotton Resources fixed the issue for that, but the Inca one is still a problem, though I realize it will most likely have to be fixed on Inca's side.

**Comment by falkreon** [comment]
C-R is now immune, yes, but the Cotton issue remains, I'll leave this open till we can track it down.
